.github/workflows/broken_link_checker.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         uses: ruzickap/action-my-broken-link-checker@v2
         with:
           url: https://evalml.alteryx.com/en/latest/index.html
-          cmd_params: '--max-connections=10 --color=always --ignore-fragments --buffer-size=8192 --exclude="(twitter|github|cloudflare)"'
+          cmd_params: '--max-connections=10 --color=always --ignore-fragments --buffer-size=8192 --exclude="(twitter|github|cloudflare|evalml\\.alteryx\\.com\\/en\\/(stable|main|v.+).*)"'
```

Comment:
Nice! Did you run this to make sure it would pass?

Reply:
Yep, ran it locally with the `muffet` cli and also edited the action to run on push so i could test it in github. https://github.com/alteryx/evalml/runs/5678469196?check_suite_focus=true

Reply:
Nice - need to look into muffet cli